extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/GrpcServerRecorder.java

```diff
@@ -529,15 +528,15 @@ private Map.Entry<Integer, Server> buildServer(Vertx vertx, GrpcServerConfigurat
             Map<String, List<String>> virtualMethodsPerService,
             GrpcContainer grpcContainer, LaunchMode launchMode) {
 
-        int port = launchMode == LaunchMode.TEST ? configuration.testPort : configuration.port;
+        int port = launchMode == LaunchMode.TEST ? configuration.testPort() : configuration.port();
 
         AtomicBoolean usePlainText = new AtomicBoolean();
```

Comment:
Where do you set this value?

Reply:
I haven't touched that part, but it is down below: 

```java
            // add Vert.x specific stuff here
            vsBuilder.useSsl(options -> {
                try {
                    usePlainText.set(applySslOptions(configuration, options));
                } catch (IOException e) {
                    throw new UncheckedIOException(e);
                }
            });
```